### DISCUSSION TITLE
how access to northwind and basic samples  pages and code demo in new version

### DISCUSSION THREAD

**Discussion by abualmazen** [serenity-is/Serenity#6761] [General]
Hi
how i can access to northwind and basic samples  pages and code demo ?
i can't fount it in new version

**Comment by edwardch** [comment]
Have a look in the common-features repo.

This demo code was split from the Serene template some time back to keep the template tidy and easier to maintain, and easier to remove the demo code from your new app.

https://github.com/serenity-is/common-features/tree/master/src/Serenity.Demo.Northwind

**Reply by abualmazen** [reply]
Thank you

**Reply by ali-h2010** [reply]
@abualmazen ,were you able to run the sample project? i am getting some dependencies missing errors. should i include the project in the serenity template or restore some packages?

**Reply by ali-h2010** [reply]
I noticed that there is a build command file but when i run it i get the following
```
Tool 'sergen' was successfully updated from version '6.7.2' to version '6.7.6' (manifest file C:\Projects\common-features-master\src\.config\dotnet-tools.json).
fatal: not a git repository (or any of the parent directories): .git
Error while restoring C:\Projects\common-features-master\build\Package.Build.props
Press any key to continue . . .
```

**Reply by ali-h2010** [reply]
@volkanceylan could you please help in running the sample? there are no steps or description of that.
